docs/cloud/azureai/tracing/index.md

```diff
@@ -0,0 +1,84 @@
+# Tracing from local to cloud
+
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../../../how-to-guides/faq.md#stable-vs-experimental).
+:::
+
+Prompt flow [tracing feature](../../../how-to-guides/tracing/index.md) enables users to trace LLM calls, functions and even LLM frameworks. Besides, with Azure extension installed, prompt flow can also log traces to an Azure ML workspace or Azure AI project, which makes it possible to share traces with your team members.
```

Comment:
what do we mean by "Azure extension"?

Reply:
the `extra_requires` part, let me search for an official name for this.

Reply:
use optional dependencies now, borrow from <https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#optional-dependencies>